### PR TITLE
Update experience-insights-dashboard.md

### PR DESCRIPTION
**Pull request by msbemba**
Added New links for all additional resources

Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9372

### CODE REVIEW

File: microsoft-365/admin/misc/experience-insights-dashboard.md [hunk 1]
```diff
@@ -90,12 +90,12 @@ Productivity Score supports the journey to digital transformation with insights
 
 ### Take training for adoption specialists
 
-In this introductory course, you'll learn the six critical elements to drive adoption of your Microsoft cloud services to deliver value to your company. This course is applicable to any size company and uses Office 365 and Microsoft Teams as the example service to create real world scenarios.
+In this introductory course, you'll learn the six critical elements to drive adoption of your Microsoft cloud services to deliver value to your company. This course is applicable to any size company and uses Office 365 and Microsoft Teams as the example service to create real world scenarios. For more information on training for adoption specialists, read, [Training for adoption specialists](https://docs.microsoft.com/learn/paths/m365-service-adoption/)
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/misc/experience-insights-dashboard.md]:
```suggestion
In this introductory course, you'll learn the six critical elements to drive adoption of your Microsoft cloud services to deliver value to your company. This course is applicable to any size company and uses Office 365 and Microsoft Teams as the example service to create real-world scenarios. For more information on training for adoption specialists, read [Use the Microsoft service adoption framework to drive adoption in your enterprise](/learn/paths/m365-service-adoption).
```

File: microsoft-365/admin/misc/experience-insights-dashboard.md [hunk 2]
```diff
@@ -90,12 +90,12 @@ Productivity Score supports the journey to digital transformation with insights
...
 
 ### Join the adoption community
 
-Welcome to the Driving Adoption Community! Connect and discuss the latest topics and best practices in driving cloud adoption. Meet and learn from peers and Microsoft Staff and stay up to date on upcoming trainings, events and our monthly Community calls.
+Welcome to the Driving Adoption Community! Connect and discuss the latest topics and best practices in driving cloud adoption. Meet and learn from peers and Microsoft Staff and stay up to date on upcoming trainings, events and our monthly Community calls. For more information read, [Adoption community](https://techcommunity.microsoft.com/t5/driving-adoption/ct-p/DrivingAdoption)
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/misc/experience-insights-dashboard.md]:
```suggestion
Welcome to the Driving Adoption Community! Connect and discuss the latest topics and best practices in driving cloud adoption. Meet and learn from peers and Microsoft Staff and stay up to date on upcoming training, events, and our monthly Community calls. For more information, read [Driving Adoption](https://techcommunity.microsoft.com/t5/driving-adoption/ct-p/DrivingAdoption).
```

File: microsoft-365/admin/misc/experience-insights-dashboard.md [hunk 3]
```diff
@@ -90,12 +90,12 @@ Productivity Score supports the journey to digital transformation with insights
...
 
 ### Use the Microsoft 365 adoption tools
 
-Use our resources to go from inspiration to execution with our productivity cloud. Get started, experiment with our services, and onboard employees at scale while being confident that you are improving the employee experience.
+Use our resources to go from inspiration to execution with our productivity cloud. Get started, experiment with our services, and onboard employees at scale while being confident that you are improving the employee experience. For more information read, [Microsoft 365 adoption tools](https://adoption.microsoft.com)
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/misc/experience-insights-dashboard.md]:
```suggestion
Use our resources to go from inspiration to execution with our productivity cloud. Get started, experiment with our services, and onboard employees at scale while being confident that you are improving the employee experience. For more information, read [Drive value with Microsoft 365 adoption tools](https://adoption.microsoft.com)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: microsoft-365/admin/misc/experience-insights-dashboard.md [hunk 4]
```diff
@@ -90,12 +90,12 @@ Productivity Score supports the journey to digital transformation with insights
...
+Welcome to the Driving Adoption Community! Connect and discuss the latest topics and best practices in driving cloud adoption. Meet and learn from peers and Microsoft Staff and stay up to date on upcoming training, events, and our monthly Community calls. For more information, read [Driving Adoption](https://techcommunity.microsoft.com/t5/driving-adoption/ct-p/DrivingAdoption).
 
 ### Use the Microsoft 365 adoption tools
 
-Use our resources to go from inspiration to execution with our productivity cloud. Get started, experiment with our services, and onboard employees at scale while being confident that you are improving the employee experience.
+Use our resources to go from inspiration to execution with our productivity cloud. Get started, experiment with our services, and onboard employees at scale while being confident that you are improving the employee experience. For more information, read [Drive value with Microsoft 365 adoption tools](https://adoption.microsoft.com)
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/misc/experience-insights-dashboard.md]:
```suggestion
Use our resources to go from inspiration to execution with our productivity cloud. Get started, experiment with our services, and onboard employees at scale while being confident that you are improving the employee experience. For more information, read [Drive value with Microsoft 365 adoption tools](https://adoption.microsoft.com).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Apologies @msbemba, one more, please.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@kwekuako it is not merged yet. 
cc: @tiburd

File: microsoft-365/admin/misc/experience-insights-dashboard.md [hunk 5]
```diff
@@ -90,12 +90,12 @@ Productivity Score supports the journey to digital transformation with insights
 
 ### Take training for adoption specialists
 
-In this introductory course, you'll learn the six critical elements to drive adoption of your Microsoft cloud services to deliver value to your company. This course is applicable to any size company and uses Office 365 and Microsoft Teams as the example service to create real world scenarios.
+In this introductory course, you'll learn the six critical elements to drive adoption of your Microsoft cloud services to deliver value to your company. This course is applicable to any size company and uses Office 365 and Microsoft Teams as the example service to create real-world scenarios. For more information on training for adoption specialists, read [Use the Microsoft service adoption framework to drive adoption in your enterprise](/learn/paths/m365-service-adoption).
```
Comment (tiburd) [inline microsoft-365/admin/misc/experience-insights-dashboard.md]:
```suggestion
In this introductory course, you'll learn the six critical elements to drive adoption of your Microsoft cloud services to deliver value to your company. This course is applicable to any size company and uses Office 365 and Microsoft Teams as the example service to create real-world scenarios. For more information about training for adoption specialists, read [Use the Microsoft service adoption framework to drive adoption in your enterprise](/learn/paths/m365-service-adoption).
```

**Pull request merged**